templates/widgets/teachers.html

```diff
@@ -1,4 +1,4 @@
 <div id="teachers_banner">
-  <object type="application/x-shockwave-flash" data="img/summer_banner.swf" height="170" width="287"></object>
+  <img src='/img/knowledge_day.gif' height="170" width="287"></object>
```

Comment:
Make sure to add patches via `git commit --amend` in order not to make any additional commits. Commits should be logically complete!